docs/devel/get-build.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+KUBE_RELEASE_BUCKET="https://storage.googleapis.com/kubernetes-release"
+KUBE_BINARY_NAME="kubernetes.tar.gz"
+
+usage() {
+	echo "usage:
+  $0 [stable|release|latest]
+
+stable:  latest stable version
+release: latest release candidate
+latest:  latest ci build to pass end to end tests"
+}
+
+if [[ $# -ne 1 ]]; then
+    usage
+    exit 1
+fi
+
+if [[ "$1" == "latest" ]]
```

Comment:
The only difference between the branches is the location of the kube version file. You should be able to move the common bits (curling it down and setting the binary tar url) outside of the if statements.